compiler/parser/parser.cpp

```diff
@@ -879,9 +880,12 @@ static DynoErrorHandler* gDynoErrorHandler = nullptr;
 
 int dynoRealizeErrors(void) {
```

Comment:
This is declared `static` in the prototype. Can we declare it `static` here as well?